ci-operator/step-registry/openshift-qe/installer/nutanix/openshift-qe-installer-nutanix-workflow.yaml

```diff
@@ -0,0 +1,15 @@
+workflow:
+  as: openshift-qe-installer-nutanix
+  steps:
+    post:
+    - chain: cucushift-installer-rehearse-nutanix-ipi-deprovision    
+    pre:
+    - chain: ipi-conf-nutanix
+    - ref: ipi-conf-osimage-nutanix
+    - ref: ipi-conf-nutanix-customized-resource
+    - chain: ipi-install
+    - ref: configure-registry-storage-nutanix
+    - chain: cucushift-installer-check
```

Comment:
You are trying to launch a simple IPI nutanix cluster? If yes, pls call `cucushift-installer-rehearse-nutanix-ipi-provision` chain directly.

Reply:
`cucushift-installer-rehearse-nutanix-ipi-zones-customized-resource-provision` support to setting replica via `COMPUTE_REPLICAS` ENV, so you can reference the chain.

Reply:
Thanks Jianlin, I will remove useless steps once I provision a nutanix clusters with 24 worker nodes. I will try your suggestion first.

Reply:
It report the error  ` /bin/bash: line 35: prism_element1_uuid: unbound variable` when I use cucushift-installer-rehearse-nutanix-ipi-zones-customized-resource-provision, I use partial of chain in cucushift-installer-rehearse-nutanix-ipi-zones-customized-resource-provision in my customized workflow